netcore/sample/HelloWorld/HelloWorld.csproj

```diff
@@ -4,6 +4,7 @@
     <OutputType>Exe</OutputType>
     <OutputPath>bin</OutputPath>
     <TargetFramework>netcoreapp3.0</TargetFramework>
+    <DebugType>full</DebugType>
```

Comment:
what's the reason for this change? netcore should use portable pdb everywhere

Reply:
unwanted change, removed